static/js/confirm_dialog.js

```diff
@@ -64,7 +64,7 @@ export function launch(conf) {
 
     const yes_button = confirm_dialog.find(".confirm_dialog_yes_button");
 
-    yes_button.html(conf.html_yes_button);
+    yes_button.html("Confirm");
```

Comment:
This should be tagged for translation using `i18n.t`, like any other user-facing string.  But also I think this is probably just wrong since we may want to allow overrides in special cases -- so `git grep html_yes_button` and change the callers to not pass this parameter, and then make "Confirm" the default if it was not passed.

See https://zulip.readthedocs.io/en/latest/translating/internationalization.html.

Reply:
Done